content/uc-doc/contributors/contributing_doc.md

```diff
@@ -2,140 +2,101 @@
 title: Contributing to the Documentation
 ---
 
-Wazo documentation is generated with GatbsyJS. The source code is
-available on GitHub at <https://github.com/wazo-platform/wazo-platform.org>
-
 ## Documentation guideline {#documentation-guideline}
 
-Here\'s the guideline/conventions to follow for the Wazo documentation.
+Here\'s the guideline/conventions to follow for the Wazo documentation. Wazo
+documentation is generated with GatbsyJS. The source code is available on GitHub
+at <https://github.com/wazo-platform/wazo-platform.org>
 
 ### Language {#language}
 
 The documentation must be written in english, and only in english.
 
-### Sections {#sections}
-
-The top section of each file must be capitalized using the following
-rule: capitalization of all words, except for articles, prepositions,
-conjunctions, and forms of to be.
-
-Correct:
-
-    The Vitamins are in My Fresh California Raisins
-
-Incorrect:
-
-    The Vitamins Are In My Fresh California Raisins
-
-Use the following punctuation characters:
-
--   `*` with overline, for \"file title\"
--   `=`, for sections
--   `-`, for subsections
--   `^`, for subsubsections
-
-There should be 2 empty lines between sections, except when an empty
-section is followed by another section.
-
-Correct:
-
-    ## Section1
-
-    Foo.
-
-
-    ## Section2
-
-    Bar.
-
-Correct:
-
-    ## Section1
-
-    Foo.
-
+### File structure {#file-structure}
 
-    .. _target:
+### File and Folders
 
-    ## Section2
+- Word should be separate by dashes (ex: `/my-folder/my-file.md`)
+- Don't use dot (`.`)
 
-    Bar.
+#### Images
 
-Correct:
+- Should be stored in `/static/images/uc-doc/`
+- Should reproduce page path `/static/images/uc-doc/page1/subpage2/myimage.jpg`
 
-    ## Section1
+#### Content
 
-    ### Subsection1
+Documentation content should be stored in `/content/uc-doc/`.
 
-    Foo.
+#### Table of Contents
 
-Incorrect:
+The table of contents is based on files and folders structure on `/content/uc-doc`.
+If you want to create a new level you must :
+- Create a new folder
+- Add an `index.md` with a list of subpages (Required to generate a clean navigation)
+- Create a new file `mypage.md`
+- Write your content inside the new file
 
-    ## Section1
+### Headings
 
-    Foo.
-
-    ## Section2
-
-    Bar.
+- The top section of each file must be capitalized using the following
+rule: capitalization of all words, except for articles, prepositions,
+conjunctions, and forms of to be.
+  ```
+  <!-- Correct -->
+  The Vitamins are in My Fresh California Raisins
+
+  <!-- Incorrect -->
+  The Vitamins Are In My Fresh California Raisins
+  ```
+- Must use `#` syntax to create a title
+- Must be H2 or lower. _(H1 titles are automatically generated by React)_
+  ```
+  ## My Title 1
+
+  ### My Subtitle 1.1
+
+  ## My Title 2
+  ```
```

Comment:
Are there any reason to indent block by 2 spaces?

Reply:
To keep the code block inside the list item.